rules/how-to-monetize-apps/rule.md

```diff
@@ -14,66 +16,87 @@ redirects:
+**Good Example:** [Microsoft 365](https://www.microsoft.com/microsoft-365) (formerly Office 365) is a suite of productivity tools that run on mobile, desktop and web. For a low recurring cost, users get access to these apps and the connected services that tie them together in the cloud.
 :::
 
-
-
+### Micro-transactions
+Micro-transactions is the practice of continually charging comparatively small amounts of money over a long period of time. Generally, micro-transactions are considered unethical because they are closely aligned with nefarious practices like [dark patterns](https://www.youtube.com/watch?v=lJUW0iZzAaQ), 'pay-to-win' or exploiting gambling addition.
+    
+For these reasons, micro-transactions are [increasingly](https://www.vg247.com/australian-government-rules-games-microtransactions-must-address-packaging) [subject](https://www.gamespot.com/articles/you-might-have-to-prove-your-age-to-buy-a-loot-box/1100-6474443/) to scrutiny from [governments](https://www.aph.gov.au/Parliamentary_Business/Committees/House/Social_Policy_and_Legal_Affairs/Onlineageverification/Report) and [regulators](https://www.gamesindustry.biz/australian-investigation-finds-loot-boxes-are-pscyholgocially-akin-to-gambling). This might be the right approach for you if:
+* You don't mind profiting from the suffering of others
+* You are morally bankrupt
+* You would consider a sacrifice of your first-born child to the Devil in exchange for wealth to be a worthwhile transaction
 
 ::: bad
-Bad example: treat all customers the same  
-:::
-
-
-::: good
-Good example: detect when a customer might leave and offer them incentives  
+`youtube: https://www.youtube.com/embed/fKK9nVLvhGM`
+**Example:** Pokemon Go, one of the most successful mobile apps of all time, combines the freemium model with micro-transactions to exploit a vulnerable subset of users. It's incredibly effective, but unethical.
 :::
 
+## Monetizing free apps
+Most free apps are designed to make money too. There are of course exceptions (for example apps created by government departments or agencies, such as the [Service NSW app](https://www.service.nsw.gov.au/service-nsw-mobile-app)), but generally most apps need to generate revenue; even the free ones. Free apps and games can still make money, and these are some of the ways that they do so.
 
+### Ad revenue
+The most common way for free apps and games to make money is through advertising. Ads are shown to users in the app or game, either as banners that are constantly there, or as 'interstitial' ads (ads that appear between levels or interactions and block the rest of the app or game until they are finished). In-app advertising is still a significant revenue stream, but is considered less-professional or less-polished for certain categories of games or apps. This might be the right approach for you if:
+* Your app is a mobile or web app (desktop users are less tolerant of in-app ads)
```

Comment:
It would be awesome if you could link to something that talks about this, it would add a huge amount of value to the point.

Reply:
Done